src/lib/plugin.js

```diff
@@ -74,6 +106,14 @@ class FiveBellsLedger extends EventEmitter2 {
     this.account = options.account
     this.username = options.username
 
+    // optional: use account@ledger.example in order to authenticate
+    // instead of using account URI + username
+    this.identifier = options.identifier
```

Comment:
Isn't `identifier` a type of credential? If so, shouldn't it be part of the `credentials` object?